server/src/Logger.hs

```diff
@@ -53,8 +70,12 @@ processLogRecords logger = go
     go = do
       cmd <- atomically $ readTBQueue (loggerQueue logger)
       case cmd of
-        LogRecord logRecord -> do
+        LogRecord logLevel logRecord -> do
           T.putStrLn logRecord
+          when (logLevel == LogError && isJust (loggerSentryService logger) ) (
+              putStrLn "Sending error message to Sentry" >>
```

Comment:
I don't think we need this in production. If sending succeeds, it was redundant, if sending fails, we'll get the output of the fallback logger configured in the sentry service.